static/js/useragent.js

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Kind of sucks we need this code in both the SDK and theme haha, but I guess that's life. At least it's not a bigger library like bowser or something